src/exchanges.js

```diff
@@ -1,4 +1,5 @@
-let Exchanges = require('pulse-publisher');
+//let Exchanges = require('pulse-publisher');
+let Exchanges = require('taskcluster-lib-pulse');
```

Comment:
Note that this needs to be `{Exchanges}` -- using object de-structuring to get one particular property of the several properties this library offers.

Reply:
sounds crazy, JS never fails to surprise lol